src/buffer_manager.ts

```diff
@@ -587,6 +594,17 @@ export class BufferManager implements Disposable, NeovimRedrawProcessable, Neovi
         return false;
     }
 
+    private findPathFromFileName(name: string): string {
+        const rootFolderPath = workspace.workspaceFolders![0].uri.fsPath;
```

Comment:
I should have checked the tests - this line fails when there is no workspace folder (brand new vscode window).